### ISSUE TITLE
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]

### ISSUE THREAD

**Issue by jenssegers** [Butterwire/serverless-plugin-monorepo#21]
When adding the `serverless-plugin-monorepo` in our yarn workspace project, we're getting this error:

```
tenants:   Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in /xxx/api/node_modules/is-promise/package.json
tenants:       at applyExports (internal/modules/cjs/loader.js:491:9)
tenants:       at resolveExports (internal/modules/cjs/loader.js:507:23)
tenants:       at Function.Module._findPath (internal/modules/cjs/loader.js:635:31)
tenants:       at Function.Module._resolveFilename (internal/modules/cjs/loader.js:953:27)
tenants:       at Function.resolve (internal/modules/cjs/helpers.js:78:19)
```

Any idea what may be causing this?

**Comment by victorlambert** [comment]
Same here. I am using `type-orm` and downgrading to `0.2.26` fix the issue.
Haven't checked deeper why yet.

**Comment by ba-amit** [comment]
Any fixes around this?

**Comment by tysoncadenhead** [comment]
This appears to be the fix: https://github.com/Butterwire/serverless-plugin-monorepo/pull/23
At least, it works on my side.

**Comment by martin-css** [comment]
Now fixed in v0.10.0
